src/main/drivers/bus_i2c_stm32f10x.c

```diff
@@ -87,6 +87,8 @@ void I2C2_EV_IRQHandler(void)
     i2c_ev_handler();
 }
 
+// I2C_DEFAULT_TIMEOUT already defined in bus_i2c.h
```

Comment:
Mmhhh... I just renamed it in another commit. Probably not what you were expecting but it fixes at least the warning. People with I2C knowledge should have a look at this, this long timeout can hide some problems.